_extensions/bluesky-comments/bluesky-comments.lua

```diff
@@ -52,27 +55,58 @@ local function getFilterConfig(meta)
     end
   end
 
-  -- Convert to JSON string
-  return quarto.json.encode(filterConfig)
+  return filterConfig
```

Comment:
Note, we return the `config` table directly because we need to access `config.profile` during render. We now call `quarto.json.encode()` directly when we inline `config` into the web component